src/lib/metrics.ts

```diff
@@ -460,30 +827,6 @@ export default class MetricsMonitor {
                         .set(stage.duration);
                 });
 
-                eventBus.on(
-                    events.STAGE_ENTERED,
-                    (entered: { stage: string; feature: string }) => {
-                        if (flagResolver.isEnabled('trackLifecycleMetrics')) {
-                            logger.info(
-                                `STAGE_ENTERED listened ${JSON.stringify(entered)}`,
-                            );
-                        }
-                        featureLifecycleStageEnteredCounter.increment({
-                            stage: entered.stage,
-                        });
-                    },
-                );
-
-                eventBus.on(
-                    events.EXCEEDS_LIMIT,
-                    ({
-                        resource,
-                        limit,
-                    }: { resource: string; limit: number }) => {
-                        exceedsLimitErrorCounter.increment({ resource, limit });
-                    },
-                );
```

Comment:
Moved up to line 485 on the right side. Now lives inside `registerPrometheusMetrics` instead of `startMonitoring`